src/main/java/ai/shreds/application/ApplicationDeleteSupplierInputPort.java

```diff
@@ -0,0 +1,23 @@
+package ai.shreds.application;
+
+import ai.shreds.adapter.AdapterDeleteSupplierResponse;
+
+/**
+ * Interface for deleting a supplier.
+ */
+public interface ApplicationDeleteSupplierInputPort {
+    /**
+     * Deletes a supplier by ID.
+     *
+     * @param id the ID of the supplier to delete
+     * @return the response after attempting to delete the supplier
+     */
+    AdapterDeleteSupplierResponse deleteSupplier(Long id);
```

Comment:
change the return type of the deleteSupplier method from AdapterDeleteSupplierResponse to void. and apply the changement to the implementation in the ApplicationSupplierService class. and the deleteSupplier method on the AdapterSupplierController class that  call it